src/components/static/Home.tsx

```diff
@@ -1,7 +1,46 @@
-import React from "react";
+import React, { useState } from "react";
 import './static.css';
+import { Tab, Icon } from "semantic-ui-react";
 
 export default function Home() {
+    const [tabHidden, setTabHidden] = useState(true)
+
+    const panes = [{
+        menuItem: 'Detailed Serosurvey Data (All)',
+        render: () => <Tab.Pane className="col-12 p-0">
```

Comment:
Maybe consider abstracting this into a method that accepts 'src' as a parameter since similar logic is used 3 times